app/_src/networking/transparent-proxy/introduction.md

```diff
@@ -0,0 +1,129 @@
+
+{{ Kuma }} uses [iptables](https://linux.die.net/man/8/iptables) and also has experimental support for [`eBPF`](#transparent-proxy-with-ebpf-experimental) to make this possible.
+
+{% tip %}
+For details on how the transparent proxy works in {{ Kuma }}, see the [Technical Overview]({{ docs }}/networking/transparent-proxy/technical-overview/).
+{% endtip %}
+
+## Kubernetes
+
+In [Kubernetes mode]({{ docs }}/introduction/architecture/#kubernetes-mode), the transparent proxy is automatically set up through the `kuma-init` container or [Kuma CNI]({{ docs }}/production/dp-config/cni/). By default, it intercepts all incoming and outgoing traffic and routes it through the `kuma-dp` sidecar container, so no changes to the application code are needed.
+
+{{ Kuma }} works smoothly with [Kubernetes DNS for Services and Pods](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) and provides its own [Kuma DNS]({{ docs }}/networking/dns/), which is especially helpful in multi-zone setups for cross-zone service discovery.
+
+In this mode, {{ Kuma }} requires the transparent proxy to be enabled, so it **cannot be turned off**.
+
+### Configuration
+{:#kubernetes-installation}
+
+If the default settings don’t fit your needs, see [Adjusting Transparent Proxy Configuration on Kubernetes]({{ docs }}/networking/transparent-proxy/configuration-on-kubernetes/) for ways to change settings and their recommended uses.
+
+{{ tproxy-config-reference-info }}
+
+## Universal
+
+Using the transparent proxy in [Universal mode]({{ docs }}/introduction/architecture/#universal-mode) makes setup easier and enables features that wouldn’t be possible otherwise. Key benefits include:
+
+- **Simplified `Dataplane` resources**: You can skip the `networking.outbound` section, so you don’t have to list each service your application connects to manually. Here’s an example without outbound entries:
+
+  ```yaml
+  type: Dataplane
+  name: {% raw %}{{ name }}{% endraw %}
+  networking:
+    address: {% raw %}{{ address }}{% endraw %}
+    inbound:
+    - port: {% raw %}{{ port }}{% endraw %}
+      tags:
+        kuma.io/service: demo-client 
+    transparentProxying:
+      redirectPortInbound: {{ tproxy.defaults.redirect.inbound.port }}
+      redirectPortOutbound: {{ tproxy.defaults.redirect.outbound.port }}
+  ```
+
+- **Easier service naming**: Use the `.mesh` [DNS domain]({{ docs }}/networking/dns/) to connect to services, like `https://service-1.mesh`, without needing `localhost` and ports from the `Dataplane` resource.
+
+- **Flexible service naming with [VirtualOutbound]({{ docs }}/policies/virtual-outbound) policy**: It lets you:
```

Comment:
I think that we should be promoting new MeshService since 2.9. Maybe we should link HostNameGenerator here instead of VirtualOutbounds for version > 2.9.x.

Reply:
addressed